cvat-ui/src/components/annotation-page/annotations-actions/annotations-actions-modal.tsx

```diff
@@ -0,0 +1,584 @@
+
+    if (type === 'select') {
+        return (
+            <Select value={value} onChange={setValue}>
+                {values.map((_value: string) => (
+                    <Select.Option key={_value} value={_value}>{_value}</Select.Option>
+                ))}
+            </Select>
+        );
+    }
+
+    const [min, max, step] = values.map((val) => +val);
+    return (
+        <InputNumber
+            value={+value}
+            min={min}
+            max={max}
+            step={step}
+            onChange={(_value: number | null) => {
+                if (typeof _value === 'number') {
+                    if (Number.isInteger(step)) {
+                        const updated = _value - (_value % step);
+                        setValue(`${clamp(updated, min, max)}`);
+                    } else {
+                        setValue(`${clamp(_value, min, max)}`);
+                    }
+                }
+            }}
+        />
+    );
+}
+
+function AnnotationsActionsModalContent(props: { onClose: () => void; }): JSX.Element {
+    const { onClose } = props;
+    const isMounted = useIsMounted();
+    const storage = getCVATStore();
+    const cancellationRef = useRef<boolean>(false);
+    const jobInstance = storage.getState().annotation.job.instance as Job;
+
+    const [state, dispatch] = useReducer(reducer, {
+        actions: [],
+        fetching: false,
+        activeAction: null,
+        progress: null,
+        progressMessage: null,
+        cancelled: false,
+        autoSaveEnabled: storage.getState().settings.workspace.autoSave,
+        jobHasBeenSaved: true,
+        frameFrom: jobInstance.startFrame,
+        frameTo: jobInstance.stopFrame,
+        actionParameters: {},
+        modalVisible: true,
+    });
+
+    useEffect(() => {
+        core.actions.list().then((list: BaseSingleFrameAction[]) => {
+            if (isMounted()) {
+                dispatch(reducerActions.setJobSavedFlag(!jobInstance.annotations.hasUnsavedChanges()));
+                dispatch(reducerActions.setAnnotationsActions(list));
+            }
+        });
+    }, []);
+
+    const {
+        actions, activeAction, fetching, autoSaveEnabled, jobHasBeenSaved,
+        progress, progressMessage, frameFrom, frameTo, actionParameters, modalVisible,
+    } = state;
+
+    return (
+        <Modal
+            closable={false}
+            width={640}
+            visible={modalVisible}
+            destroyOnClose
+            footer={null}
+            afterClose={onClose}
+            className='cvat-action-runner-content'
+        >
+            <Row>
+                <Col span={24} className='cvat-action-runner-info'>
+                    <Alert
+                        message={(
+                            <div>
+                                <Text>Actions allow executing certain algorithms on </Text>
+                                <Text strong>
+                                    <a href={config.FILTERS_GUIDE_URL}>filtered</a>
```

Comment:
Please add target blank, i dont think we want to close cvat on opening this link

Reply:
Added